leo/commands/mod.rs

```diff
@@ -14,53 +14,125 @@
 // You should have received a copy of the GNU General Public License
 // along with the Leo library. If not, see <https://www.gnu.org/licenses/>.
 
-pub mod add;
-pub use self::add::*;
+use crate::context::{get_context, Context};
+use anyhow::Result;
```

Comment:
nit: please organize the imports here into two sections - leo specific crates & external crates